### PR TITLE
feat: add Sentry (VO-334)

### PR DESCRIPTION
**Pull request by acezard**
```
### 🔧 Tech

* Add sentry to monitor crash event
```

### CODE REVIEW

File: src/targets/browser/index.jsx [hunk 1]
```diff
@@ -155,6 +157,18 @@ export const initApp = () => {
...
       isPublic: isPublic
     })
+    Sentry.init({
+      dsn: 'https://16c26a60d9019eea9d9a9775573e3765@errors.cozycloud.cc/73',
+      environment: process.env.NODE_ENV,
+      release: manifest.version,
+      integrations: [
+        new CaptureConsole({ levels: ['error'] }) // We also want to capture the `console.error` to, among other things, report the logs present in the `try/catch`
+      ],
+      tracesSampleRate: 1,
```
Comment (cballevre) [inline src/targets/browser/index.jsx:167]: It should be changed to 0.1 to avoid polluting the server 🙂

File: src/targets/browser/index.jsx [hunk 2]
```diff
@@ -155,6 +157,18 @@ export const initApp = () => {
       cozyClient: client,
       isPublic: isPublic
     })
+    Sentry.init({
+      dsn: 'https://16c26a60d9019eea9d9a9775573e3765@errors.cozycloud.cc/73',
+      environment: process.env.NODE_ENV,
+      release: manifest.version,
+      integrations: [
+        new CaptureConsole({ levels: ['error'] }) // We also want to capture the `console.error` to, among other things, report the logs present in the `try/catch`
```
Comment (cballevre) [inline src/targets/browser/index.jsx:165]: Sry I read too fast the first time but if you want to use `withSentryReactRouterV6Routing` you need also to enable `reactRouterV6BrowserTracingIntegration` : 

imports : 
```
import {
  useLocation,
  useNavigationType,
  createRoutesFromChildren,
  matchRoutes
} from 'react-router-dom'
```

code for sentry init :
```
integrations: [
    Sentry.reactRouterV6BrowserTracingIntegration({
      useEffect: React.useEffect,
      useLocation,
      useNavigationType,
      createRoutesFromChildren,
      matchRoutes,
    }),
...
],
```

if you like more explanation there is this documentation : https://docs.sentry.io/platforms/javascript/guides/react/features/react-router/

Comment (acezard) [reply src/targets/browser/index.jsx]: @cballevre updated

**Review by cballevre — APPROVED** [review]
Nice 👍

**Pull request merged**